homeassistant/components/mqtt/strings.json

```
@@ -183,17 +183,17 @@
         },
         "data_description": {
           "discovery": "Option to enable MQTT automatic discovery.",
-          "discovery_prefix": "The prefix of configuration topics the MQTT interation will subscribe to.",
+          "discovery_prefix": "The prefix of configuration topics the MQTT integration will subscribe to.",
           "birth_enable": "When set, Home Assistant will publish an online message to your MQTT broker when MQTT is ready.",
-          "birth_topic": "The MQTT topic where Home Assistant will publish a birth message.",
-          "birth_payload": "The birth message that is published when MQTT is ready and connected.",
-          "birth_qos": "The quality of service of the birth message that is published when MQTT is ready and connected",
-          "birth_retain": "When set, Home Assistant will retain the birth message published to your MQTT broker.",
-          "will_enable": "When set, Home Assistant will ask your broker to publish a will message when MQTT is stopped or looses the connection to your broker.",
-          "will_topic": "The MQTT topic your MQTT broker will publish a will message to.",
-          "will_payload": "The message your MQTT broker will publish when the connection is lost.",
-          "will_qos": "The quality of service of the will message that is published by your MQTT broker.",
-          "will_retain": "When set, your MQTT broker will retain the will message."
+          "birth_topic": "The MQTT topic where Home Assistant will publish a `birth` message.",
+          "birth_payload": "The `birth` message that is published when MQTT is ready and connected.",
+          "birth_qos": "The quality of service of the `birth` message that is published when MQTT is ready and connected",
+          "birth_retain": "When set, Home Assistant will retain the `birth` message published to your MQTT broker.",
+          "will_enable": "When set, Home Assistant will ask your broker to publish a `will` message when MQTT is stopped, or looses the connection to your broker.",
+          "will_topic": "The MQTT topic your MQTT broker will publish a `will` message to.",
+          "will_payload": "The message your MQTT broker `will` publish when the connection is lost.",
```

```suggestion
          "will_payload": "The message your MQTT broker `will` publish when the connection is lost.",
```
...when the connection is lost or was stopped? Above those are mentioned as two distinct cases.

---

I updated the suggestion, is that better?

---

thank you!